public/static/locales/en/form-builder-responses.json

```diff
@@ -43,54 +43,86 @@
       "noResponsesToDownload": "There are no responses available to download."
     },
     "notifications": {
-      "downloading":"Downloading",
       "downloadComplete": "Download complete",
-      "downloadingXFiles": "Downloading {{fileCount}} files",
-      "downloadXSelectedResponses": "Download {{size}} selected responses",
+      "downloadingXFiles": "Downloading {{fileCount}} files"
+    },
+    "errors": {
       "trySelectingLessFilesHeader": "Only {{max}} responses can be downloaded at a time.",
       "trySelectingLessFiles":"You have {{max}} responses selected. Uncheck at least 1 response before downloading.",
       "atLeastOneFileHeader": "At least one file must be selected to download.",
       "atLeastOneFile": "Try selecting at least one file to download.",
       "errorDownloadingFilesHeader":"Error downloading the selected files.",
       "errorDownloadingFiles":"Try again in a few minutes. If the problem persists contact support."
     },
```

Comment:
"errorDownloadingFilesLink" missing?

Reply:
Good find. I should also double check that error since I wrote it I think